pallets/gear/src/lib.rs

```diff
@@ -1030,49 +1035,28 @@ pub mod pallet {
                 T::DebugInfo::remap_id();
             }
 
+            let lazy_pages_enabled =
+                cfg!(feature = "lazy-pages") && lazy_pages::try_to_enable_lazy_pages();
+
             while QueueProcessingOf::<T>::allowed() {
                 if let Some(dispatch) = QueueOf::<T>::dequeue()
                     .unwrap_or_else(|e| unreachable!("Message queue corrupted! {:?}", e))
```

Comment:
I think we can reduce one nesting level by doing like this
```suggestion
                let dispatch = match QueueOf::<T>::dequeue()
                    .unwrap_or_else(|e| unreachable!("Message queue corrupted! {:?}", e)) 
                {
                    Some(dispatch) => dispatch,
                    None => break,
                };  
```

Reply:
Yeah, but should we?

It reduces amount of tabs, but for me decreases readability, which isn't good as well